python/pyspark/sql/dataframe.py

```diff
@@ -1511,13 +1511,18 @@ def inSet(self, *cols):
     isNull = _unary_op("isNull", "True if the current expression is null.")
     isNotNull = _unary_op("isNotNull", "True if the current expression is not null.")
 
-    def alias(self, alias):
+    def alias(self, *alias):
         """Return a alias for this column
 
         >>> df.select(df.age.alias("age2")).collect()
         [Row(age2=2), Row(age2=5)]
         """
-        return Column(getattr(self._jc, "as")(alias))
+
+        if len(alias) == 1:
+            return Column(getattr(self._jc, "as")(alias[0]))
+        else:
+            sc = SparkContext._active_spark_context
+            return Column(getattr(self._jc, "as")(_to_seq(sc, list(alias))))
```

Comment:
There are two tests for this case already:
 - https://github.com/apache/spark/pull/6107/files#diff-7c2fe8530271c0635fb99f7b49e0c4a4R131
 - https://github.com/apache/spark/pull/6107/files#diff-f5295f69bfbdbf6e161aed54057ea36dR180